packages/material-ui-styles/src/propsToClassKeys/propsToClassKeys.js

```diff
@@ -0,0 +1,32 @@
+import MuiError from '@material-ui/utils/macros/MuiError.macro';
+
+// TODO: remove this once the capitalize method is moved to the @material-ui/utils package
+export function capitalize(string) {
+  if (typeof string !== 'string') {
+    throw new MuiError('Material-UI: capitalize(string) expects a string argument.');
+  }
+
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function isEmpty(string) {
+  return string.length === 0;
+}
+
+export default function propsToClassKey(props) {
```

Comment:
Could you document what this is supposed to do and what it should be used for? If this has only one callsite inline it. No need to create a standalone module on introduction.

Reply:
Added description. This is used in two places so far, so that's why I decided to extract it as a separate module.

Reply:
Hm some tests would be nice. Should be more descriptive than the JSDOC.

Reply:
Added 👍